src/main/java/chess/domain/Team.java

```diff
@@ -0,0 +1,17 @@
+package chess.domain;
+
+public enum Team {
+
+    BLACK, WHITE;
```

Comment:
tmi 개인취향이긴 한데 이렇게 쓰는 걸 선호해요ㅎㅎ
```suggestion
    BLACK,
    WHITE,
    ;
```

이렇게 되면 새로운 것이 추가되었을 때 딱 추가된 값만 보이기도 해서 좋더라고용